app/decorators/organization_decorator.rb

```diff
@@ -4,6 +4,10 @@ class OrganizationDecorator < Draper::Decorator
 
   delegate_all
 
+  def org_type_string
+    organization.supplier? ? 'supplier' : 'buyer'
+  end
```

Comment:
Is there a better name for something like this?

Reply:
in app/models/organization.rb
```ruby 
ORG_TYPE_SUPPLIER = 'supplier'
ORG_TYPE_BUYER = 'buyer'
```
and refer to those.

Reply:
I would actually refactor this to use [simple_enum](https://github.com/lwe/simple_enum) and just be able to say this in the mailers or views:
```
@organization.org_type # buyer
```

Reply:
Another pro is then we get a clear path on how to do localization for that string.

Reply:
Reviewed this PR on a call with Weston. I concur that it's best to refactor the organization type to use SimpleEnum.

Reply:
@ivar @thermistor I could use a bit of help here. Are you imagining I'd do a migration to update all single-character org_types in database (i.e. 'B') to their enum string (i.e. 'buyer')?

I've tried this (and a bunch of permutations) with supporting code but when running specs I get `Validation failed: Org type can't be blank`.
```
# In modes/Org
validates :org_type, presence: true
as_enum :org_type, [buyer: 'B', supplier: 'S', market: 'M', admin: 'A'], source: :org_type, map: :string, accessor: :whiny

# In factories
org_type Organization.org_types[:supplier]
```

I'm not sure that simple_enum lets me map existing database string values to a string as label. Or I'm missing something basic.

Reply:
Yeah, to use `simple_enum` you would have to remap the db values to integers. Alternatively, because it's in the decorator you could do:
```
# in organization_decorator.rb
def org_type
   h.t(o.org_type, scope: [:decorators,:organization, :org_type])
end
# in config/locales/decorators/organization.yml
en:
  decorators:
    organization:
       org_type:
         S: supplier
         B: buyer